sdk/translation/azure-ai-translation-text/samples/sample_text_translation_translate.py

```diff
@@ -0,0 +1,307 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_text_translation_translate.py
+
+DESCRIPTION:
+    This file contains sample snippets for the Text Translation service.
+
+USAGE:
+    python sample_text_translation_translate.py
+
+    Set the text translation endpoint environment variables with your own value before running the samples:
+        
+        1) AZURE_TEXT_TRANSLATION_ENDPOINT - the endpoint to your Text Translation resource.
+        Note: the endpoint must be formatted to use the custom domain name for your resource:
+        https:\\<NAME-OF-YOUR-RESOURCE>.cognitiveservices.azure.com\
+                    
+    The create_text_translation_client_with_credential call requires additional variables:
+        2) AZURE_TEXT_TRANSLATION_APIKEY - the API key to your Text Translation resource.
+        3) AZURE_TEXT_TRANSLATION_REGION - the Azure Region of your Text Translation resource.
+"""
+
+from azure.core.exceptions import HttpResponseError
+from azure.ai.translation.text.models import (InputTextItem, TextType, ProfanityAction, ProfanityMarker)
```

Comment:
You dont need parenthesis around the imports unless you're making them multiline, this applies to multiple sample files.